stable/traefik/templates/deployment.yaml

```diff
@@ -62,16 +62,18 @@ spec:
             cpu: "{{ .Values.cpuLimit }}"
             memory: "{{ .Values.memoryLimit }}"
         readinessProbe:
-          tcpSocket:
-            port: 80
+          httpGet:
+            path: /ping
```

Comment:
`/ping` is a liveliness endpoint. Not sure it is appropriate for use in readiness in this case.

Reply:
I think http check is better than tcp. The tcp check is shallow readiness check in this case.